src/main/java/eu/xenit/gradle/alfrescosdk/internal/tasks/AmpConfigSourceDirectorySet.java

```diff
@@ -0,0 +1,16 @@
+package eu.xenit.gradle.alfrescosdk.internal.tasks;
+
+import org.gradle.api.file.SourceDirectorySet;
+import org.gradle.api.model.ObjectFactory;
+
+import java.text.MessageFormat;
+
+public final class AmpConfigSourceDirectorySet {
+    public static final String NAME = "alfrescoAmpConfig";
+    
+    public static SourceDirectorySet create(String name, ObjectFactory objectFactory) {
```

Comment:
Do we need separate factories for this?
It is pretty much a duplication of `AmpWebSourceDirectorySet`, and both do not really do much.